spec/services/booking_request_creator_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe BookingRequestCreator do
+  let(:prisoner_step) {
+    PrisonerStep.new(
+      prison_id: 1,
+      first_name: 'Oscar',
+      last_name: 'Wilde',
+      date_of_birth: Date.new(1980, 12, 31),
+      number: 'a1234bc'
+    )
+  }
+
+  let(:visitors_step) {
+    VisitorsStep.new(
+      first_name: 'Ada',
+      last_name: 'Lovelace',
+      date_of_birth: Date.new(1970, 11, 30),
+      email_address: 'ada@test.example.com',
+      phone_no: '01154960222'
+    )
+  }
+
+  let(:slots_step) {
+    SlotsStep.new(
+      option_1: '2015-01-02T09:00/10:00',
+      option_2: '2015-01-03T09:00/10:00',
+      option_3: '2015-01-04T09:00/10:00'
+    )
+  }
+
```

Comment:
Also, SlotsStep sounds like something you do when the Zumba class is oversubscribed :smiling_imp: